packages/kubernetes-1.22/Cargo.toml

```diff
@@ -14,8 +14,8 @@ path = "pkg.rs"
 package-name = "kubernetes-1.22"
 
 [[package.metadata.build-package.external-files]]
-url = "https://distro.eks.amazonaws.com/kubernetes-1-22/releases/19/artifacts/kubernetes/v1.22.17/kubernetes-src.tar.gz"
-sha512 = "8a71caf9c490022fe3865bae21b49c56046598004dba524c12cf9ff7954585fb9f97c903539fd491229c0200437462aad46e388c34dd66d22dfa7461a47dfac4"
+url = "https://distro.eks.amazonaws.com/kubernetes-1-22/releases/22/artifacts/kubernetes/v1.22.17/kubernetes-src.tar.gz"
+sha512 = "c7b4929375f1cceaf7ccafc12c566e1be57dcf95e093a166e65ccb8933f9ebd86e349e0f1f498b3723127661ff9abf76e6e0124eb6033c869c71f8e124424697"
```

Comment:
Is this SHA-512 checksum correct?

I'm getting something different.
```
$ curl -OL https://distro.eks.amazonaws.com/kubernetes-1-22/releases/22/artifacts/kubernetes/v1.22.17/kubernetes-src.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 34.6M  100 34.6M    0     0  39.9M      0 --:--:-- --:--:-- --:--:-- 39.9M

$ sha512sum kubernetes-src.tar.gz 
2f44e34a350cee52b3e26e5939d4543919d57341868f35a5551f04cc7326f2d0a0ab5a9bcd04d1980ef74c4a4e8f28faae4ef8810d801d9ee8aca6328cbd836d  kubernetes-src.tar.gz
```

Reply:
I'll look into this. I think our script might be wrong